roles/nova-data/tasks/main.yml

```diff
@@ -23,6 +23,10 @@
 - name: probe nbd
   modprobe: name=nbd state=present
 
+- name: disable SMT for ppc64le
+  command: "ppc64_cpu --smt=off"
+  when: ansible_architecture == "ppc64le"
```

Comment:
Is there any way to detect when this action results in a change? I'd like to limit the number of "changes" we show each run, so that we someday could have some meaningful idea of what all changed when updating a cluster.